rekuho/src/lib/Rekuho.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts" context="module">
+  import kanagawa from "./kanagawa.js"
+  import type { Theme } from "./Theme"
+
+  const websocket = async () => {
+    try {
+      const ws = new WebSocket("ws://localhost:6969")
+      const cmd = {
+        method: "key_pressed",
+        params: {
+          key: "X",
+          modifiers: [],
+        },
+      }
+
+      await new Promise((resolve) => ws.addEventListener("open", () => resolve(null)))
+      ws.send(JSON.stringify(cmd))
+    } catch (e) {
+      console.log(e)
+    }
+  }
+
+  export let theme: Theme = {
+    font: {
+      family: "monospace",
+      weight: "normal",
+      size: "20px",
+    },
+    text_offset: 10,
+    gutter: {
+      background: kanagawa.sumiInk0,
+      width: 50,
+    },
+
+    font_color: kanagawa.fujiWhite,
+    primary_cursor_color: kanagawa.fujiWhite,
+    editor_background: kanagawa.sumiInk1,
```

Comment:
I'm out of the loop on the color theme choice? Shouldn't this be part of Theme.ts or something?

Reply:
Probably. Again, prototypical.

Reply:
Moved to `Theme.ts` :D